promotion/replication/replication.ipynb

```diff
@@ -6,110 +6,355 @@
    "metadata": {},
```

Comment:
<p>With mpec and the other notebook we should write a few more comments or markup cells on what we do here and whats the main difference to the other replication notebook. This notebook and the simulation are more tutorials on how to use ruspy, while the other is a complete replication of a specific paper!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/ruspy/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>